.env-example

```diff
@@ -0,0 +1,15 @@
+NODE_ENV=production
+MYSQL_HOST=localhost
+MYSQL_PORT=3306
+MYSQL_DB=pinning_service
+MYSQL_USER=root
+MYSQL_PASSWORD=root
```

Comment:
Do not check password, better way is delete all the values and describe the meanings in `README`